org.eclipse.scanning.test/src/org/eclipse/scanning/test/scan/nexus/NexusTest.java

```diff
@@ -173,13 +176,13 @@ public static void setServices() throws Exception {
 	@Before
 	public void createFile() throws IOException {
 		output = File.createTempFile("test_nexus", ".nxs");
-		output.deleteOnExit();
+//		output.deleteOnExit();
 	}
 	
 	@After
 	public void deleteFile() {
 		try {
-			output.delete(); 
+//			output.delete();

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)